shared/cmd/flags.go

```diff
@@ -66,9 +66,15 @@ var (
 	// BootstrapNode tells the beacon node which bootstrap node to connect to
 	BootstrapNode = &cli.StringFlag{
 		Name:  "bootstrap-node",
-		Usage: "The address of bootstrap node. Beacon node will connect for peer discovery via DHT.  Multiple nodes can be separated with a comma",
+		Usage: "The address of bootstrap node. Beacon node will connect for peer discovery via DHT.  Multiple nodes can be separated with a comma.  Ignored if the bootstrap-node-file flag is used.",
 		Value: "enr:-Ku4QMKVC_MowDsmEa20d5uGjrChI0h8_KsKXDmgVQbIbngZV0idV6_RL7fEtZGo-kTNZ5o7_EJI_vCPJ6scrhwX0Z4Bh2F0dG5ldHOIAAAAAAAAAACEZXRoMpD1pf1CAAAAAP__________gmlkgnY0gmlwhBLf22SJc2VjcDI1NmsxoQJxCnE6v_x2ekgY_uoE1rtwzvGy40mq9eD66XfHPBWgIIN1ZHCCD6A",
 	}
+	//BootStrapNodeFile identifies a YAML file that lists bootstrap nodes
```

Comment:
```suggestion
	//BootStrapNodeFile identifies a YAML file that lists bootstrap nodes.
```

Reply:
I included it in my most recent commit.  I didn't want a commit just for the period.  Thanks!

Reply:
It's all good! Just fyi we squash all the commits for the merge : )

Reply:
Nvm.  I guess not...